docs/release-notes/snapcraft-8-7.rst

```diff
@@ -137,6 +116,42 @@ The following issues have been resolved in Snapcraft 8.7:
 - `craft-application#620`_ The remote builder help suggested using a nonextistent
   ``--status`` option.
 
+.. _release-8.7.1:
```

Comment:
I wonder if these will be too generic and result in collisions later. What about something more specific, like:

```
_release-notes-fixes-8.7.1
```